nansat/tests/nansat_test_base.py

```diff
@@ -0,0 +1,54 @@
+# ------------------------------------------------------------------------------
+# Name:         nansat_test_base.py
+# Purpose:      Basic class for Nansat tests
+#
+# Author:       Anton Korosov
+#
+# Created:      20.03.2018
+# Copyright:    (c) NERSC
+# Licence:      This file is part of NANSAT. You can redistribute it or modify
+#               under the terms of GNU General Public License, v.3
+#               http://www.gnu.org/licenses/gpl-3.0.html
+# ------------------------------------------------------------------------------
+from __future__ import print_function, absolute_import, division
+import os
+import sys
+import unittest
+import tempfile
+import pythesint
+
+if sys.version_info.major == 2:
```

Comment:
I thought you got rid of that in some PR in the past. I remember that I made a review